pinery-miso/src/main/resources/ca/on/oicr/pinery/lims/miso/queryAllOrderSamples.sql

```diff
@@ -2,7 +2,7 @@ SELECT o.sequencingOrderId orderId
         ,lib.NAME libraryId 
         ,bc1.sequence barcode 
         ,bc2.sequence barcode_two 
-        ,sp.paired paired 
+        ,sp.readLength = sp.readLength2 paired 
```

Comment:
Doesn't this mean that 10X runs will be described as single-end? I don't think that's accurate.

Reply:
We define them as single ended right now, so it's consistent with the legacy. I don't know what actually looks at information. CellRanger and BCL2FASTQ look at the bases mask, so I don't know if this matters. I could make it `readLength2 != 0`, which I guess is more accurate, but might cause all the 10X runs to go stale.

Reply:
Half of our runs have `paired = 1` so I think we're already losing on consistency...
`paired` is used to make the `readLength`, which is independent of the `runBasesMask`. It doesn't look like `pinery_ius` even includes the `read_length`, so I don't think it'll be an analysis issue.

Reply:
Okay, fixed. Even if `pinery_ius` doesn't use it, it still becomes part of the hash in the LIMS key, so we'll just have to see.